testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -605,103 +585,132 @@ class TestGuessFormat(object):
         ('XYZ', mda.topology.XYZParser.XYZParser, mda.coordinates.XYZ.XYZReader),
     ]
     if six.PY2:
-        # DCD, LAMMPS, and TRZ are not supported on Python 3 yet
+        # TRZ is not supported on Python 3 yet
         formats += [
-            ('DATA', mda.topology.LAMMPSParser.DATAParser,
-             mda.coordinates.LAMMPS.DATAReader),
-            ('DCD', None, mda.coordinates.DCD.DCDReader),
-            ('LAMMPS', None, mda.coordinates.LAMMPS.DCDReader),
             ('TRZ', None, mda.coordinates.TRZ.TRZReader),
         ]
     # list of possible compressed extensions
     # include no extension too!
     compressed_extensions = ['.bz2', '.gz']
 
-    def _check_get_ext(self, f, fn):
+    @pytest.mark.parametrize('extention', [format[0].upper() for format in formats] +
```

Comment:
what happens if you choose another name. A static analyzer might confuse this with the built-in format.